tests/manifest.csv

```diff
@@ -1,62 +1,63 @@
-test065,"blank node - double circle of 3 (0-2-1, reversed)",,,TRUE
-test066,"blank node - double circle of 3 (1-0-2, reversed)",,,TRUE
-test067,"blank node - double circle of 3 (1-2-0, reversed)",,,TRUE
-test068,"blank node - double circle of 3 (2-1-0, reversed)",,,TRUE
-test069,"blank node - double circle of 3 (2-0-1, reversed)",,,TRUE
-test070,dataset - isomorphic default and iri named,Isomorphic graphs in default and IRI named graph,,TRUE
-test071,dataset - isomorphic default and node named,Isomorphic graphs in default and blank node named graph,,TRUE
-test072,dataset - shared blank nodes,Blank nodes shared in default and named graph,,TRUE
-test073,dataset - referencing graph name,Default graph with blank node shared with graph name,,TRUE
\ No newline at end of file
+test,name,comment,approval,rdfc10,rdfc10map
+test001,simple id,,,TRUE,FALSE
+test002,duplicate property iri values,,,TRUE,FALSE
+test003,bnode,,,TRUE,TRUE
+test004,bnode plus embed w/subject,,,TRUE,TRUE
+test005,bnode embed,,,TRUE,TRUE
+test006,multiple rdf types,,,TRUE,FALSE
+test008,single subject complex,,,TRUE,FALSE
+test009,multiple subjects - complex,,,TRUE,FALSE
+test010,type,,,TRUE,FALSE
+test011,type-coerced type,,,TRUE,FALSE
+test013,"type-coerced type, cycle",,,TRUE,FALSE
+test014,check types,,,TRUE,FALSE
+test016,blank node - dual link - embed,,,TRUE,TRUE
+test017,blank node - dual link - non-embed,,,TRUE,TRUE
+test018,blank node - self link,,,TRUE,TRUE
+test019,blank node - disjoint self links,,,TRUE,FALSE
+test020,blank node - diamond,,,TRUE,TRUE
+test021,blank node - circle of 2,,,TRUE,FALSE
+test022,blank node - double circle of 2,,,TRUE,FALSE
+test023,blank node - circle of 3,,,TRUE,FALSE
+test024,blank node - double circle of 3 (0-1-2),,,TRUE,FALSE
+test025,blank node - double circle of 3 (0-2-1),,,TRUE,FALSE
+test026,blank node - double circle of 3 (1-0-2),,,TRUE,FALSE
+test027,blank node - double circle of 3 (1-2-0),,,TRUE,FALSE
+test028,blank node - double circle of 3 (2-1-0),,,TRUE,FALSE
+test029,blank node - double circle of 3 (2-0-1),,,TRUE,FALSE
+test030,blank node - point at circle of 3,,,TRUE,TRUE
+test033,disjoint identical subgraphs (1),,,TRUE,FALSE
+test034,disjoint identical subgraphs (2),,,TRUE,FALSE
+test035,reordered w/strings (1),,,TRUE,FALSE
+test036,reordered w/strings (2),,,TRUE,FALSE
+test038,"reordered 4 bnodes, reordered 2 properties (1)",,,TRUE,FALSE
+test039,"reordered 4 bnodes, reordered 2 properties (2)",,,TRUE,FALSE
+test040,reordered 6 bnodes (1),,,TRUE,FALSE
+test043,literal with language,,,TRUE,FALSE
+test044,evil (1),,,TRUE,FALSE
+test045,evil (2),,,TRUE,FALSE
+test046,evil (3),,,TRUE,FALSE
+test047,deep diff (1),,,TRUE,TRUE
+test048,deep diff (2),,,TRUE,TRUE
+test053,@list,,,TRUE,TRUE
+test054,t-graph,,,TRUE,FALSE
+test055,simple reorder (1),,,TRUE,TRUE
+test056,simple reorder (2),,,TRUE,TRUE
+test057,unnamed graph,,,TRUE,TRUE
+test058,unnamed graph with blank node objects,,,TRUE,TRUE

```

Comment:
```suggestion
test058,unnamed graph with blank node objects,,,TRUE,FALSE
```
I think test058 should be removed from rdfc10map tests because it has ambiguous (duplicated) `issued identifiers map`s.

*related manifest files (ttl, jsonld, ...) should also be modified

Reply:
Done.